projects/plugins/jetpack/modules/subscriptions/subscribe-modal/class-jetpack-subscribe-modal.php

```diff
@@ -62,6 +62,19 @@ public function enqueue_assets() {
 		if ( $this->should_user_see_modal() ) {
 			wp_enqueue_style( 'subscribe-modal-css', plugins_url( 'subscribe-modal.css', __FILE__ ), array(), JETPACK__VERSION );
 			wp_enqueue_script( 'subscribe-modal-js', plugins_url( 'subscribe-modal.js', __FILE__ ), array( 'wp-dom-ready' ), JETPACK__VERSION, true );
+
+			/**
+			 * Filter how many milliseconds a user must scroll for until the Subscribe Modal appears.
+			 *
+			 * @module subscriptions
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @param int 300 Time in milliseconds for the Subscribe Modal to appear upon scrolling.
+			 */
+			$load_time = absint( apply_filters( 'jetpack_subscribe_modal_load_time', 300 ) );
+
+			wp_localize_script( 'subscribe-modal-js', 'Jetpack_Subscriptions', array( 'loadTime' => $load_time ) );
```

Comment:
Nitpick but would `modalLoadTime` help as more descriptive name?

Let's also just check with @Automattic/jetpack-vulcan in case there's any Jetpack-blocks specific way or the standard plugin way like this is fine.

Reply:
Makes sense - all done!